frontend/lodh-fall/lodh.css

```
@@ -0,0 +1,2226 @@
+
+  color: rgba(255, 255, 255, 0.75);
+  font-size: 0.7rem;
+  letter-spacing: 0.16em;
+}
+
+.scroll-arrow {
+  font-size: 1.5rem;
+  animation: lodhBounce 1.6s infinite;
+}
+
+@keyframes lodhBounce {
```

Stylelint (17.14.0)

[error] Expected keyframe name "lodhBounce" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)